cmd/spire-server/cli/entry/create.go

```diff
@@ -223,16 +227,17 @@ func (CreateCLI) newConfig(args []string) (*CreateConfig, error) {
 	f.StringVar(&c.RegistrationUDSPath, "registrationUDSPath", util.DefaultSocketPath, "Registration API UDS path")
 	f.StringVar(&c.ParentID, "parentID", "", "The SPIFFE ID of this record's parent")
 	f.StringVar(&c.SpiffeID, "spiffeID", "", "The SPIFFE ID that this record represents")
-	f.IntVar(&c.Ttl, "ttl", 3600, "A TTL, in seconds, for any SVID issued as a result of this record")
+	f.IntVar(&c.Ttl, "ttl", 3600, "The lifetime, in seconds, for SVIDs issued based on this registration entry")
 
 	f.StringVar(&c.Path, "data", "", "Path to a file containing registration JSON (optional)")
 
-	f.Var(&c.Selectors, "selector", "A colon-delimeted type:value selector. Can be used more than once")
+	f.Var(&c.Selectors, "selector", "A colon-delimited type:value selector. Can be used more than once")
 	f.Var(&c.FederatesWith, "federatesWith", "SPIFFE ID of a trust domain to federate with. Can be used more than once")
 
 	f.BoolVar(&c.Node, "node", false, "If set, this entry will be applied to matching nodes rather than workloads")
 	f.BoolVar(&c.Admin, "admin", false, "If set, the SPIFFE ID in this entry will be granted access to the Registration API")
 	f.BoolVar(&c.Downstream, "downstream", false, "A boolean value that, when set, indicates that the entry describes a downstream SPIRE server")
+	f.Int64Var(&c.EntryExpiry, "entryExpiry", 0, "An expiry, from epoch in millis, for the resulting registration entry to be pruned")
```

Comment:
Hmm, I think I'd prefer seconds since epoch, instead of millis. We aren't going to be guaranteeing millisecond precision (or even second for that matter) for expiration and millis is a bit unwieldy. It also aligns more closely with other expiration-like things in the system (like ttl's, attested node expiry, etc).

In either case, we need to make sure to update the datastore.proto commentary on the field to include the expected unit of the value. I had just assumed it was seconds since epoch when doing that review. I'm sorry I didn't bring it up then :(

Reply:
:'(
will fix